packages/plugin-snowplow/src/androidMain/kotlin/ly.iterative.itly.snowplow/SnowplowPlugin.kt

```diff
@@ -0,0 +1,57 @@
+/**
+ * Android SnowplowPlugin
+ */
+package ly.iterative.itly.snowplow
+
+import ly.iterative.itly.*
+
+import com.snowplowanalytics.snowplow.Snowplow;
+import com.snowplowanalytics.snowplow.controller.TrackerController
+import com.snowplowanalytics.snowplow.event.SelfDescribing
+import com.snowplowanalytics.snowplow.payload.SelfDescribingJson
+
+actual class SnowplowPlugin actual constructor(
+    options: SnowplowOptions
+) : Plugin(ID) {
+    companion object {
+        @JvmField
+        val ID = "snowplow"
+    }
+
+    private var config: SnowplowOptions = options
+    private lateinit var logger: Logger
+    private var snowplow: TrackerController? = null
+
+    val client: TrackerController?
+        get() = this.snowplow
+
+    override fun load(options: PluginLoadOptions) {
+        logger = options.logger
+        logger.debug("[plugin-snowplow] load")
+
+        if (config.tracker != null) {
+            this.snowplow = config.tracker
+        }
+        else {
+            this.snowplow = Snowplow.getDefaultTracker()
```

Comment:
Not sure we can use `Snowplow.getDefaultTracker()` here. I think at the very least the user needs to pass in a `url` on SnowplowOptions to use for the tracker creation.